### PR TITLE
Schema for AgriParcel data model

### PR DESCRIPTION
**Pull request by fdelavega**
Schema for AgriParcel data model

### CODE REVIEW

File: specs/AgriFood/AgriGreenhouse/example-normalized-ld.jsonld [hunk 1]
```diff
@@ -0,0 +1,79 @@
+{
+    "@context": [
+        "https://forge.etsi.org/gitlab/NGSI-LD/NGSI-LD/raw/master/coreContext/ngsi-ld-core-context.json",
+        "https://raw.githubusercontent.com/GSMADeveloper/NGSI-LD-Entities/master/examples/Agri-Greenhouse-context.jsonld"
+    ],
+    "id": "urn:ngsi-ld:AgriGreenhouse:df72dc57-1eb9-42a3-88a9-8647ecc954b4",
+    "type": "AgriGreenhouse",
```
Comment (jmcanterafonseca) [inline specs/AgriFood/AgriGreenhouse/example-normalized-ld.jsonld]: be careful, this AgriGreenhouse not AgriParcel!!!!

**Comment by samgh96** [comment]
I think this build fails because the reference of location is not properly set in the reference common schema (http://json-schema.org/geojson/geometry.json returns 404).

File: specs/AgriFood/AgriParcel/schema.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "$id": "https://raw.githubusercontent.com/Ficodes/dataModels/agriDataModels/specs/AgriFood/AgriParcel/schema.json",
```
Comment (jmcanterafonseca) [inline specs/AgriFood/AgriParcel/schema.json]: use the FIWARE URL

File: specs/AgriFood/AgriParcel/example-normalized-ld.jsonld [hunk 3]
```diff
@@ -0,0 +1,86 @@
+{
+    "@context": [
+        "https://forge.etsi.org/gitlab/NGSI-LD/NGSI-LD/raw/master/coreContext/ngsi-ld-core-context.json",
```
Comment (jmcanterafonseca) [inline specs/AgriFood/AgriParcel/example-normalized-ld.jsonld]: official ETSI URI

File: specs/AgriFood/AgriParcel/example-normalized-ld.jsonld [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    "agriCrop": {
+        "type": "Relationship",
+        "object": "urn:ngsi-ld:AgriCrop:36021150-4474-11e8-a721-af07c5fae7c8"
+    },
+    "cropStatus": {
+        "type": "Property",
+        "value": "seeded"
+    },
+    "lastPlantedAt": {
+        "type": "Property",
```
Comment (jmcanterafonseca) [inline specs/AgriFood/AgriParcel/example-normalized-ld.jsonld]: encode Date Time properly

File: specs/AgriFood/AgriParcel/example-normalized.json [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+    },
+    "agriCrop": {
+        "type": "Relationship",
+        "value": "urn:ngsi-ld:AgriCrop:36021150-4474-11e8-a721-af07c5fae7c8"
+    },
+    "cropStatus": {
+        "value": "seeded"
+    },
+    "lastPlantedAt": {
+        "value": "2016-08-23T10:18:16Z"
```
Comment (jmcanterafonseca) [inline specs/AgriFood/AgriParcel/example-normalized.json]: type: DateTime missing

**Pull request merged**